ftplugin/verilog_systemverilog.vim

```diff
@@ -44,7 +44,7 @@ if exists("loaded_matchit")
     \ '\<case\>\|\<casex\>\|\<casez\>:\<endcase\>,' .
     \ '`if\(n\)\?def\>:`elsif\>:`else\>:`endif\>,' .
     \ '\<module\>:\<endmodule\>,' .
-    \ '\<if\>:\<else\>,' .
+    \ '\(else\s\+\)\@<!\<if\>:\<else\>\s\+\<if\>:\(`\)\@<!\<else\>\(\s\+if\)\@!,' .
```

Comment:
OK. I'll test it a few days later.
Maybe I can find a solution in modifying matchit.
I think that `else if` can be recognized as parallel case. (The condition is similar.)
```
unique if (sel==2'b00)
  out = ...;
else if (sel==2'b01)
  out = ...;
else if (sel==2'b10)
  out = ...;
else
  out = ...;
```
Not parallel case:
```
priority if (rst)
  register <= 32'd0;
else
begin
  unique if (cond==2'b00)
    register <= 32'd0;
  else if (cond==2'b01)
    register <= register+32'd1;
  else 
    register <= register;
end
```
Verilog doesn't restrict designers to use which usage.
I think that two usages can increase readability and reduce too many indents.